### ISSUE TITLE
FEATURE REQUEST: Chocolatey package

### ISSUE THREAD

**Issue by AIVAS001** [mitchcapper/KPEntryTemplates#44]
It would be nice if a Chocolatey package was available. Especially if from the official developer.
In theory this would mean the package release would correspond more closely with the release of the plugin both in version and date.

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
Yes, please releases Releases on Repository of Chocolatey packages?
Hello, this is not an issue in itself. Just an suggestion. Would be nice to have a auto update feature for this primary security applications plugins from trusted source. The sha hash of PLRGX can be verified by installation automatically optionally if specified.  _For the lazy ones :) and_ to fix incompatibility issue with future of keepass with plugin by getting all in one place for example.

I don't know if you have the spare time to maintain in parallel the "packages". Which basically would contains a script to URL of github latest release asset. 

If you may want to take a look here if you don't know already:

[keepass packages](https://chocolatey.org/packages?q=keepass&moderatorQueue=&prerelease=true&sortOrder=package-download-count)

There are many plugins and packages count is growing.

**Comment by mitchcapper** [comment]
@AIVAS001 @4-FLOSS-Free-Libre-Open-Source-Software 

Alright done, it is awaiting moderation right now but its up: https://chocolatey.org/packages/keepass-kpentrytemplates

**Comment by 4-FLOSS-Free-Libre-Open-Source-Software** [comment]
This is great, thanks for adding to repo.
